Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rust:1.81
+WORKDIR /app
+
+COPY . .
```

Comment:
I'm not 100% certain that I need to copy anything other than the `entrypoint.sh` script into the container.

Reply:
Agree that you probably only need `entrypoint.sh`